src/LogNormal/LogNormal.sol

```diff
@@ -74,14 +88,26 @@ contract LogNormal is PairStrategy {
         (reserves, totalLiquidity, params) =
             abi.decode(data, (uint256[], uint256, LogNormalParams));
 
+        if (params.mean < MIN_WIDTH || params.mean > MAX_MEAN) {
```

Comment:
`MIN_WIDTH` should be `MIN_MEAN` here.

Reply:
Fixed here https://github.com/primitivefinance/DFMM/pull/119/commits/2c700a62315e5fd373b13757e69dd7c255c2ed7d.